package/cleanup-list

```diff
@@ -37,7 +38,8 @@ def parse_repo(dbpath):
             pkgid = tarinfo.name
             desc_tar = repodb.getmember(os.path.join(pkgid, "desc"))
             desc_file = repodb.extractfile(desc_tar)
-            desc_file = io.TextIOWrapper(io.BytesIO(desc_file.read()), encoding="UTF-8")
+            desc_file = io.TextIOWrapper(io.BytesIO(
+                desc_file.read()), encoding="UTF-8")
```

Comment:
For the next PR, please leave out the line length changes. I know we don't have a set up any linting configuration however this PR is about typing not pep8 :)